client/clientmanager/src/main/java/io/mosip/registration/clientmanager/spi/audit/AuditManagerSerivceImpl.java

```diff
@@ -0,0 +1,166 @@
+package io.mosip.registration.clientmanager.spi.audit;
```

Comment:
Impl classes should be added directly under package: io.mosip.registration.clientmanager.service